api-server/api-server-common/src/storage/storage_api/mod.rs

```diff
@@ -82,19 +89,25 @@ impl Delegation {
         &self.balance
     }
 
+    pub fn next_nonce(&self) -> &AccountNonce {
+        &self.next_nonce
+    }
+
     pub fn add_pledge(&self, rewards: Amount) -> Self {
```

Comment:
Missed that in previous PR, but `pledge` is not a correct term for delegation. We use delegate staking / spend share in context of delegation. Or add_delegation_balance alternatively